wgsl/index.bs

```diff
@@ -8603,6 +8610,30 @@ path: syntax/location_attr.syntax.bs.include
 
 </table>
 
+<pre class=include>
+path: syntax/blend_src_attr.syntax.bs.include
+</pre>
+
+<table class='data builtin'>
+  <caption><dfn noexport dfn-for="attribute">`blend_src`</dfn> Attribute</caption>
```

Comment:
I would prefer `blend_source`.

Reply:
Since the WebGPU spec already spells this `src` (as in `"src"`, `"src-alpha"`, etc. extended to `"src1"` etc.) we [decided](https://github.com/gpuweb/gpuweb/issues/4283#issuecomment-1834451453) to use that spelling here too.